frontend/javascripts/oxalis/view/help_modal.tsx

```diff
@@ -0,0 +1,96 @@
+import { sendHelpEmail, updateNovelUserExperienceInfos } from "admin/admin_rest_api";
+import { Modal, Input, Alert } from "antd";
+import { setActiveUserAction } from "oxalis/model/actions/user_actions";
+import { OxalisState } from "oxalis/store";
+import React, { CSSProperties, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+function HelpButton() {
+  const [isModalOpen, setModalOpen] = useState(false);
+
+  const dispatch = useDispatch();
+  const activeUser = useSelector((state: OxalisState) => state.activeUser);
+
+  const discardButton = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+    // prevent the modal from also being shown
+    e.stopPropagation();
+    
+    if (!activeUser) return null;
+    if (activeUser && activeUser.novelUserExperienceInfos.hasDiscardedHelpButton) return null;
```

Comment:
should logged-out users see the help button and be able to send emails? we won't know their email to reply though, right? if we support this somehow, the close-button should be hidden, though, since it doesn't do anything.

Reply:
Logged out users should not see the help button:
1. We don't have an email to respond to
2. there is not much you can do in logged out state anyway. you likely be browsing the landing page
  2.1 alternatively, you can be logged out for viewing a public dataset/annotation. That is a read-only interaction anyway and would not expect someone to ask questions